GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+
+Selenium IDE:
+* [@corevo](https://github.com/corevo)
+* [@manoj9788](https://github.com/manoj9788)
+* [@shs96c](https://github.com/shs96c)
+
+Selenium:
+* [@adamgoucher](https://github.com/adamgoucher)
+* [@andreastt](https://github.com/andreastt)
+* [@AutomatedTester](https://github.com/AutomatedTester)
+* [@barancev](https://github.com/barancev)
+* [@cgoldberg](https://github.com/cgoldberg)
+* [@corevo](https://github.com/corevo)
+* [@davehunt](https://github.com/davehunt)
+* [@ddavison](https://github.com/ddavison)
+* [@dfabulich](https://github.com/dfabulich)
+* [@diemol](https://github.com/diemol)
+* [@DominikDary](https://github.com/DominikDary)
+* [@freynaud](https://github.com/freynaud)
+* [@illicitonion](https://github.com/illicitonion)
+* [@jarib](https://github.com/jarib)
+* [@jimevans](https://github.com/jimevans)
+* [@jleyba](https://github.com/jleyba)
+* [@jlipps](https://github.com/jlipps)
+* [@joshbruning](https://github.com/joshbruning)
+* [@juangj](https://github.com/juangj)
+* [@julianharty](https://github.com/julianharty)
+* [@krosenvold](https://github.com/krosenvold)
+* [@lmtierney](https://github.com/lmtierney)
+* [@lukeis](https://github.com/lukeis)
+* [@mach6](https://github.com/mach6)
+* [@mtscout6](https://github.com/mtscout6)
+* [@nirvdrum](https://github.com/nirvdrum)
+* [@p0deje](https://github.com/p0deje)
+* [@santiycr](https://github.com/santiycr)
+* [@sevaseva](https://github.com/sevaseva)
+* [@shs96c](https://github.com/shs96c)
+* [@titusfortner](https://github.com/titusfortner)
+* [@tourdedavetner](https://github.com/tourdedave)
+* [@twalpole](https://github.com/twalpole)
+
+Selenium Site & Docs:
+* [@ahunsberger](https://github.com/ahunsberger)
+* [@ddavison](https://github.com/ddavison)
+* [@diemol](https://github.com/diemol)
+* [@lukeis](https://github.com/lukeis)
+* [@manoj9788](https://github.com/manoj9788)
+
+Docker-Selenium:
+* [@ddavison](https://github.com/ddavison)
+* [@diemol](https://github.com/diemol)
+* [@elgalu](https://github.com/elgalu)
+* [@kayabendroth](https://github.com/kayabendroth)
+* [@mtscout6](https://github.com/mtscout6)
+* [@WillAbides](https://github.com/WillAbides)
+
+### Technical Leadership Committee (TLC)
+
+The technical decisions and roadmap of the Selenium project are governed by
+a Technical Leadership Committee (TLC) which is responsible for a high-level
+technical guidance of the project.
+
+The TLC has final authority over this project including:
+
+* Technical direction
+* Repository hosting
+* Contribution policy (shared with the PLC)
+
+TLC seats are not time-limited. There is no fixed size of the TLC. The TLC
+should be of such a size as to ensure adequate coverage of important areas
+of expertise balanced with the ability to make decisions efficiently.
+
+The TLC may add additional members to the TLC by a standard TLC motion.
+
+A TLC member may be removed from the TLC by voluntary resignation, or by a stand
+ard TLC motion.
```

Comment:
```suggestion
ard TLC or PLC motion.
```
Maybe?

Reply:
I think it is a good idea, I added that.